src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
what if the user does not have the font Roboto installed?

it would be nice to consider starting a discussion with the design team to define fallback functions for those cases.